realm/realm-library/src/main/java/io/realm/HandlerController.java

```diff
@@ -265,7 +287,7 @@ private void updateAsyncEmptyRealmObject() {
     }
 
     /**
-     * This method calls all registered listeners on Realm, RealmResults and RealmObjects.
+     * This method calls all registered listeners on Realm, RealmResults and RealmObjects and async transactions.
```

Comment:
Listeners for Realm, RealmResults and RealmObjects, and callbacks for async transactions.